lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -134,294 +134,233 @@ public function render_passwordless_method() {
     <?php
   }
 
-  public function render_jwt_auth_integration() {
-    $v = absint( $this->options->get( 'jwt_auth_integration' ) );
-
-    echo $this->render_a0_switch( "wpa0_jwt_auth_integration", "jwt_auth_integration", 1, 1 == $v );
-?>
-    <div class="subelement">
-      <span class="description"><?php echo __( 'This will enable the JWT Auth\'s Users Repository override.', 'wp-auth0' ); ?></span>
-    </div>
-  <?php
+  public function render_jwt_auth_integration( $args ) {
+    $this->render_switch( $args[ 'label_for' ], $args[ 'opt_name' ] );
+    $this->render_field_description( __( 'This will enable the JWT Auth Users Repository override', 'wp-auth0' ) );
```

Comment:
Previous string had a `'s` after `JWT`. Is this intentional? and btw, what does this string mean?

Reply:
Yes, intentional. Sounded awkward. 

JWT Auth is an unsupported plugin we put out a while ago. So this is saying if that plugin is installed and this option is on, use the "user repository" from the plugin.